Lib/exception_group.py

```diff
@@ -0,0 +1,198 @@
+
+import sys
+import textwrap
+import traceback
+
+
+class TracebackGroup:
```

Comment:
Our original plan involved something like making TracebackGroup be a generalization of Traceback, but following this exercise I don't think that makes sense. I actually don't think we need TracebackGroup as a separate class. 

1. It doesn't have a life without the ExceptionGroup because it doesn't know about the nested exceptions (only their IDs), so you can't have something like print_tb for TracebackGroups.

2. It is coupled with the ExceptionGroup so we will never do something like eg.with_traceback_group(tg).

3. The information in the TracebackGroup is actually a mirror of the __traceback__ of the exceptions in the group, so it's redundant. 

I think we should just remove it.

The traceback tree of an ExceptionGroup eg consists of the root->eg path in eg.__traceback__ and then splits into the child nodes which are the trees on the nested exceptions (so the __traceback__ of each nested exception, and so on recursively).